uw-frame-components/staticFeeds/sample_notifications.json

```diff
@@ -12,14 +12,14 @@
 	  {
 		"id"     : 3,
 		"groups" : ["Everyone"],
-		"title"  : "Checkout these cool buttons ->.",
+		"title"  : "Checkout these cool buttons",
```

Comment:
Since messing with this file was nit-picky in the first place, I appreciate this